src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeParse.cs

```diff
@@ -5227,7 +5227,7 @@ private static string Hex(char c)
                 return "\\u" + ((int)c).ToString("x4", CultureInfo.InvariantCulture);
         }
 
-        private static void Trace(string s)
+        private static void Trace(string _)
```

Comment:
This means that one has to both uncomment the trace statement and change the argument name to enable the tracing.

Reply:
re-enabled instead